src/MineOnlineConfig.java

```diff
@@ -0,0 +1,87 @@
+import java.io.File;
+
+public class MineOnlineConfig {
+    private boolean newFile = false;
+    private PropertiesFile propertiesFile;
+
+    public MineOnlineConfig(File file) {
+        if (!file.exists()) {
+            newFile = true;
+            file.getParentFile().mkdirs();
+        }
+        propertiesFile = new PropertiesFile(file.getAbsolutePath());
+        write();
+        propertiesFile.save();
+    }
+    public void write() {
```

Comment:
Please keep the following options in server.properties:
- max-players
- online-mode
- whitelist